scripts/third_party_size_check_test.py

```diff
@@ -54,8 +54,9 @@ def mock_print(msg: str) -> None:
 
         os.symlink('/dummy_dir/file1.py', 'dummy_dir/symlink_file.py')
 
-        self.dummy_file_object = open(
-            'dummy_dir/file1.py', 'r', encoding='utf-8')
+        with open(
+            'dummy_dir/file1.py', 'r', encoding='utf-8') as dummy_file_object:
+            self.dummy_file_object = dummy_file_object
```

Comment:
I don't think this is going to work. When we try to use `self.dummy_file_object` elsewhere in the tests, won't it be closed?

Reply:
Good point. I've changed it back to the non context manager `open` and added a lint ignore for this line.